src/nxdoc/web-ui/web-ui-tutorials/web-ui-bulk-actions.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Bulk Actions in Web UI'
+review:
+    comment: ''
+    date: '2021-09-07'
+    status: ok
+toc: true
+labels:
+    - nuxeo-web-ui
+    - bulk
+tree_item_index: 490
+---
+
+## Enabling Bulk Actions
+
+{{#> callout type='info'}}
+Bulk actions leverage the [Bulk Action Framework]({{page page='bulk-action-framework'}}). Make sure your [architecture]({{page page='nuxeo-reference-architectures'}}) complies with it.
+{{/callout}}
+
+Bulk actions are available since LTS 2021. To prevent bringing any breaking change through an update, you need to enable this feature explicitly by adding the following property to your [nuxeo.conf]({{page page='configuration-parameters-index-nuxeoconf'}}) file:
```

Comment:
```suggestion
Bulk actions were introduced to LTS 2021 with Web UI 3.0.6. To prevent bringing any breaking change through an update, you need to enable this feature explicitly by adding the following property to your [nuxeo.conf]({{page page='configuration-parameters-index-nuxeoconf'}}) file:
```